.github/workflows/ci.yml

```diff
@@ -10,17 +10,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [ '7.2', '7.3', '7.4', '8.0' ]
+        php: [ '8.0', '8.1' ]
         strategy: [ 'highest' ]
-        sf_version: ['']
+        sf_version: ['5.*', '6.*']
         include:
             - php: 7.4
               strategy: 'lowest'
+            - php: 7.4
+              sf_version: '5.*'
+            - php: 8.1
+              strategy: 'lowest'
             - php: 7.3
               sf_version: '3.*'
             - php: 7.3
               sf_version: '4.*'
-
+            - php: 7.3
+              sf_version: '5.*'
+              
```

Comment:
Hm, don't we want to test on PHP 7.2 as well? Because we do support it in composer.json, see https://github.com/php-translation/common/blob/6db7ada54e106621b77d82f12accd745accdd1a4/composer.json#L12

Reply:
@rvanlaak maybe at least trigger one build on the LTS Sf 4.4 & PHP 7.2 ?

Reply:
Ah my bad, missed that. You are absolutely right.